### ISSUE TITLE
Remember the last saved location

### ISSUE THREAD

**Issue by arcturus2** [constellation-app/constellation#14]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Running the latest version of Constellation
	* Attached the ***Support Package*** via `Help > `Support Package`
    * Checked the FAQs: https://github.com/constellation-app/constellation/faq
    * Checked that your issue isn't already filed: https://github.com/constellation-app/constellation/issues
    * Checked that there is not already a module that provides the described functionality: https://github.com/constellation-app/constellation/repositories-catalog

### Description

When you save a graph, the initial directory Constellation will open is the systems "home" directory so this can be for example `C:\Users\{username}` on Windows and `/home/{username}` on Linux. If you select a different location like the Desktop, subsequent `File -> Save As` do open the Desktop folder.

However, when you close and open Constellation again it will default back to the "home" directory of the user.

### Steps to Reproduce

1. File -> Save As
2. Select the Desktop
3. Click Save
4. Close Constellation
5. Open Constellation
6. File -> New Graph
7. File -> Save As

**Expected behaviour:** The Save As dialog should open the Desktop folder

**Actual behaviour:** The Save As dialog shows the "home" directory

**Reproduces how often:** 100%

### Additional Information

An eligant solution to this problem should include the NetBeans Preferences which is the approach we use to save state between restarts.

**Comment by serpens24** [comment]
Inspected this issue with a fresh master build and changes have been merged in.

Repeated steps listed in issue and found the save location from previous run was used. I then did the following:

1. Turned off 'Remember Save Location' in Options->CONSTELLATION
2. Opened a new graph and then selected Save As... from the File menu
**The save dialog reverted to my home directory**
3. Shutdown and restarted Constellation
4. Opened a new graph and then selected Save As... from the File menu
**The save dialog reverted to my home directory**
5. Turned on 'Remember Save Location' in Options->CONSTELLATION
6. Selected Save As... from the File menu
** The save dialog used the last saved directory - prior to me unchecking 'Remember Save Location'. I guess the question is, do we think this should have occured, or whether it should have been set to my home directory, I'm happy as it is, although there is probably an argument that when you turn OFF 'Remember Save Location'  that the previously stored path value is removed from the preferences file ... effectively resetting the history.**

Can this issue be closed out @arcturus2 or should there be an additional change made in line with point 6 above ?
